### DISCUSSION TITLE
Ability to take screenshots

### DISCUSSION THREAD

**Discussion by berger20** [rafaelvcaetano/melonDS-android#1287] [Ideas]
Currently I couldn't find a way to take a screenshot beyond an OS (android) screenshot.
This means you get the UI in the way, and it's oversized and the strange resolution of the phone instead of the standard DS (or multiplied) resolution.

Screenshots are good for memories, remembering hints/puzzles, or to report bugs.
The places I checked for such a feature, was in the Pause menu, and in the customisable layout buttons.

**Comment by cyruzwings** [comment]
why not use Power Down+Power for android

**Reply by berger20** [reply]
That's what I've been doing, but it gets the UI in the way, and the image is stretched instead of pixel perfect.

You could say the same for any app on Windows, but most emulators still have an in app screenshot availablity.

**Comment by frillfish** [comment]
+1, I really would like my screenshots to be crisp without having to shrink down the screens in the layout settings.
